data.md

```diff
@@ -343,15 +344,18 @@ Bitwise logical operators are lifted from the integer versions.
 -   `#nBits` shifts in $N$ ones from the right.
 -   `#nBytes` shifts in $N$ bytes of ones from the right.
 -   `_<<Byte_` shifts an integer 8 bits to the left.
+-   `_>>Byte_` shifts an integer 8 bits to the right.
 
 ```k
     syntax Int ::= #nBits  ( Int )  [function]
                  | #nBytes ( Int )  [function]
                  | Int "<<Byte" Int [function]
+                 | Int ">>Byte" Int [function]
```

Comment:
Is this used anywhere?

Reply:
1. >>Byte : yes it's used in proofs.
2. We had 2 versions of #bytes before, and this was redundant. Now we have only one and a code sugar (in verified-smart-contracts) in place of the removed one.